src/app/types/User.ts

```diff
@@ -40,6 +40,8 @@ export interface User extends TableData, CreatedUpdatedProps {
   eulaAcceptedVersion: number;
   eulaAcceptedOn: Date;
   billingData: BillingUserData;
+  technical: boolean;
+  projectedFields: string[];
```

Comment:
projectedFields can be put in the global type named TableData which will be used everywhere

Reply:
you mean the global type TableData defined in Table.ts (line 22):
export interface **TableData** extends AuthorizationActions {
?

Reply:
AuthorizationActions is even better indeed :-)